django/contrib/auth/mixins.py

```diff
@@ -79,9 +79,12 @@ def get_permission_required(self):
             perms = self.permission_required
         return perms
 
+    def check_for_permission(self, user, perms):
```

Comment:
Can you drop the user argument, please. You already gave all information in `self....`.